app/vector_icons/convert_svg_to_icon.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2023 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import optparse
+import subprocess
+import os
+
+
+def process(input_file, output_file):
```

Comment:
Could we replace `-` with `_` here and remove the change in the patch file below by any chance?

Reply:
I was trying to do that originally, but `outputs` needs to be statically defined in `action_foreach` so I wasn't able to give it the correct output name :/

Reply:
Oh, I see, {{source_name_part}} is constant. Thanks for explaining!

Reply:
Yeah, it's a bit of a pain, tbh :laughing: